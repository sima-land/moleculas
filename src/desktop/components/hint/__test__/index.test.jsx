import React from 'react';
import { mount } from 'enzyme';
import { Hint } from '..';

describe('<Hint />', () => {
  it('should renders correctly', () => {
    const wrapper = mount(
      <Hint className='additional-class'>Hello, world!</Hint>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
