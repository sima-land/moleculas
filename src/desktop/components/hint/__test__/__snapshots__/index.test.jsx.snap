// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hint /> should renders correctly 1`] = `
<ForwardRef
  className="additional-class"
>
  <div
    className="root additional-class"
    data-testid="hint"
  >
    <div
      className="body"
    >
      Hello, world!
    </div>
  </div>
</ForwardRef>
`;
